### PR TITLE
fix(designer): Fix Comments on nodes

### PR DESCRIPTION
**Pull request by Eric-B-Wu**
With the pinning of panel, comments were broken:
Before:
commentbefore

After:
commentfix

### CODE REVIEW

File: .prettierrc
Comment (ccastrotrejo) [inline .prettierrc:1]: Do we need this?

**Review by ccastrotrejo — APPROVED** [review]

File: .prettierrc
Comment (Eric-B-Wu) [reply .prettierrc:1]: I think the default prettier setting are different from the one we currently use in the repo, so every time I do format, the whole file gets formatted differently. I'm thinking we have this config file, so we're all on the same page

Comment (ccastrotrejo) [reply .prettierrc:1]: Sounds good, thanks for doing it!

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (ek68794998) [inline .prettierrc:1]: Shouldn't this be a Biome config?

**Comment by ek68794998** [comment]
Thanks for catching this!

File: .prettierrc (hunk 1 above)
Comment (Eric-B-Wu) [reply .prettierrc:1]: hmm i wonder why my biome config is not catching it locally then

**Pull request merged**